udata/templates/organization/display.html

```diff
@@ -140,7 +140,7 @@
     {% endfor %}
 
     {% if org.members %}
-    <tab id="members" header="{{ _('Members') }}">
+    <tab id="members" header="{{ ngettext('%(num)d member', '%(num)d members', org.members | length) }}">
```

Comment:
AFAIK, metrics are computed via cron tasks using rabbitmq so if you want to test that part you probably need to tweak your local instance. Not straightforward 😞